src/configuration.rs

```diff
@@ -102,6 +106,11 @@ pub struct WebUI {
     pub enabled: bool,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct ServerSentEvent {
```

Comment:
This should probably be named `ServerSentEvents` instead. Kind of pedantic though, and it's hidden from the user anyway so this would be fine too.

Reply:
Yes you right!